app/experimenter/experiments/api/v5/types.py

```diff
@@ -390,7 +390,10 @@ class Meta:
 
     def resolve_feature_config(self, info):
         if self.feature_configs.exists():
-            return self.feature_configs.all().order_by("slug").first()
+            return sorted(
+                self.feature_configs.all(),
+                key=lambda feature_config: (feature_config.slug),
+            )[0]
```

Comment:
Oh right yes, good, it was just hitting this subquery multiple times.  Looks like that's all that was needed for the landing page, great catch.